.github/workflows/tekton_task_tests.yaml

```
@@ -22,8 +22,26 @@ jobs:
           dir_names: "true"
           dir_names_max_depth: "3"
       - name: Show changed dirs
+        id: show-changed-dirs
+        shell: bash
         run: |
           echo ${{ steps.changed-dirs.outputs.all_changed_files }}
+          trustedArtifactsBasedTasks=()
+          for dir in ${{ steps.changed-dirs.outputs.all_changed_files }}
+          do
+            echo "$dir"
+            baseDir=$(basename "${dir}")
+            taskFile="${dir}/${baseDir}.yaml"
+            if [ -f "${taskFile}" ]; then
+              ociStageParam=$(yq '.spec.params[] | select(.name == "ociStorage") | .name' "${taskFile}")
+              if [ ! -z "${ociStageParam}" ]; then
+                echo "Found a trusted artifacts compatible task: ${taskFile}"
+                trustedArtifactsBasedTasks+=($dir)
+              fi
+            fi
```

Maybe this is better? If you prefer yours that's fine.
```suggestion
            if [ -f "${taskFile}" ] \
                && [[ $(yq '[.spec.params[] | select(.name == "ociStorage")] | length' \
                  tasks/managed/apply-mapping/apply-mapping.yaml) -gt 0 ]]
            then
              echo "Found a trusted artifacts compatible task: ${taskFile}"
              trustedArtifactsBasedTasks+=($dir)
            fi
```

---

used yours. thanks